package.xml

```diff
@@ -6,4 +6,5 @@
   <maintainer email="69460051+4c3y@users.noreply.github.com">4c3y</maintainer>
   <license>BSD</license>
   <buildtool_depend>catkin</buildtool_depend>
+  <depend>roscpp</depend>
```

Comment:
in the off case, where catkin is used but ros isnt, this will fail.

but I guess at the moment we seldomly have this case.

Reply:
I'll leave it for now and fix it if necessary.